src/neoxp/Extensions/ExpressNodeExtensions.cs

```diff
@@ -396,6 +405,36 @@ public static async Task<(RpcNep17Balance balance, Nep17Contract token)> GetBala
             throw new Exception("invalid script results");
         }
 
+        public static async Task<List<string>> GetNFTAsync(this IExpressNode expressNode, UInt160 accountHash, string asset)
+        {
+            var assetHash = await expressNode.ParseAssetAsync(asset).ConfigureAwait(false);
+
+            using var sb = new ScriptBuilder();
+            sb.EmitDynamicCall(assetHash, "tokensOf", accountHash);
+
+            var result = await expressNode.InvokeAsync(sb.ToArray()).ConfigureAwait(false);
+            var stack = result.Stack;
+            var list = new List<string>();
+            try
+            {
+                if (result.State != VMState.FAULT
+                        && result.Stack.Length >= 1
+                        && result.Stack[0] is InteropInterface interop
+                        && interop.GetInterface<object>() is IIterator iterator)
+                {
+                    while (iterator.Next())
+                    {
+                        list.Add(iterator.Value(null).GetString());
+                    }
+                }
+            }
+            catch (Exception)
+            {
+                throw new Exception("invalid script results");
```

Comment:
Can we get a better error output? Something indicating that the contract isn't an NFT or missing a method